website/content/cli/usage/data/data-sources.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Data sources
+sidebar_position: 1
+description: This page provides useful information on dealing with different data
+  vendors when using the OpenBB Platform CLI. It outlines how to select a default data source,
+  acquire API keys, and switch the data vendor using specific commands, all in an
+  effort to streamline and improve the user's experience.
+keywords:
+- Terminal
+- CLI
+- data vendors
+- API keys
+- data sources
+- FinancialModelingPrep
+- Polygon
+- AlphaVantage
+- EODHD
+- YahooFinance
+- source
+- stocks/fa/income
+- changing data source
+- Default data source
+- /sources
+- get --cmd
+- set --cmd
+---
+
+import HeadTitle from '@site/src/components/General/HeadTitle.tsx';
+
+<HeadTitle title="Data sources - Data - Usage | OpenBB Platform CLI Docs" />
+
+import TutorialVideo from '@site/src/components/General/TutorialVideo.tsx';
+
+<TutorialVideo
+  youtubeLink="https://www.youtube.com/embed/cvSwG96Yf4o?si=oswcJYUH51F206Hu"
+  videoLegend="Short video on where the data comes from"
+/>
+
+:::note
+Note that the commands and menus may vary.
+:::
+
+## Relationship With Data Vendors
```

Comment:
We don't need this section, and we can just say something like: "data sources available rely on the OpenBB Platform Provider extensions you have installed."